src/ftrackWidget.ts

```diff
@@ -161,18 +210,21 @@ function onDocumentClick(event) {
 /**
  * On document keydown forward to parent application.
  */
-function onDocumentKeyDown(event) {
+function onDocumentKeyDown(event: KeyboardEvent) {
   // Ignore events when focus is in an textarea/input/contenteditable.
-  const tagName = event.target.tagName.toLowerCase();
+  const target = event.target;
+  if (!target || !(target instanceof HTMLElement)) return;
```

Comment:
I also realised that `!(target instanceof HTMLElement)` is an implicit null check as well, but I feel it's probably more readable doing that null check "twice".